.github/workflows/pr-test.yml

```
@@ -0,0 +1,44 @@
+          script: |
+            const pull_number = context.payload.pull_request.number;
+            const title = context.payload.pull_request.title;
+            const updated_title = `❌[BUILD FAIL] ${title}`;
+      
+            await github.rest.pulls.createReview({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number,
+              body: "Test failed: 코드를 다시 확인해주세요.",
+              event: "REQUEST_CHANGES"
+            });
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)